docs/jwst/pipeline/calwebb_spec2.rst

```diff
@@ -40,54 +40,58 @@ TSO exposures. The instrument mode abbreviations used in the table are as follow
+| :ref:`assign_wcs <assign_wcs_step>`                      | |c| | |c| | |c| | |c| | |c| | |c| |  |c| | |c|  |  |c|   | |c| |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`background <background_step>`                      | |c| | |c| | |c| | |c| |     | |c| |  |c| | |c|  |  |c|   |     |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`imprint <imprint_step>`                            |     | |c| | |c| |     |     |     |      |      |        |     |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`msaflagopen <msaflagopen_step>`                    |     | |c| | |c| |     |     |     |      |      |        |     |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`extract_2d <extract_2d_step>`\ :sup:`1`            | |c| | |c| |     |     |     |     |      | |c|  |  |c|   | |c| |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`srctype <srctype_step>`\ :sup:`1`                  | |c| | |c| | |c| | |c| | |c| | |c| |  |c| |      |        | |c| |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`master_background <master_background_step>`        |     | |c| |     |     |     |     |      |      |        |     |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`wavecorr <wavecorr_step>`                          | |c| | |c| |     |     |     |     |      |      |        |     |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`flat_field <flatfield_step>`\ :sup:`1`             | |c| | |c| | |c| | |c| | |c| | |c| |  |c| | |c|  |  |c|   | |c| |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`straylight <straylight_step>`                      |     |     |     |     |     | |c| |      |      |        |     |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`fringe <fringe_step>`                              |     |     |     |     |     | |c| |      |      |        |     |
++----------------------------------------------------------+-----+-----+-----+-----+-----+-----+------+------+--------+-----+
+| :ref:`residual_fringe <residual_fringe_step>` \ :sup:`2` |     |     |     |     |     | |c| |      |      |        |     |
```

Comment:
I thought I remembered hearing that `residual_fringe` had to be done after the `photom` step, because it was setup to work on flux-calibrated data?

Reply:
you are right. I fixed the table. It is in the correct place when it is called in the pipeline